docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -0,0 +1,458 @@
+***********************
+Using Sawtooth with SGX
+***********************
+
+.. note::
+
+    These instructions have been tested on Ubuntu 16.04 only.
+
+Prerequisites
+=============
+
+.. _bios-update:
+
+BIOS Update
+-----------
+
+.. Important::
+
+    You may need to update your BIOS with a security fix before running
+    Hyperledger Sawtooth with PoET. Affected versions and instructions for
+    updating can be found on
+    `Intel's website <https://security-center.intel.com/advisory.aspx?intelid=INTEL-SA-00076&languageid=en-fr>`_.
+    If you're running an affected version, you must update the BIOS
+    to ensure that these installation instructions work correctly.
+
+You can verify the BIOS version after the machine has booted by running:
+
+.. code-block:: console
+
+    $ sudo lshw| grep -A5 *-firmware
+         *-firmware
+              description: BIOS
+              vendor: Intel Corp.
+              physical id: 0
+              version: BNKBL357.86A.0050.2017.0816.2002
+              date: 08/16/2017
+
+
+Certificate File
+----------------
+
+The configuration process requires an SGX certificate file in PEM format
+(.pem), which you will need before continuing.
```

Comment:
You can install SGX dependencies without the SPID cert. Might be better to just list this below around line ~169.

Reply:
To create your own service provider certificate follow these directions:
https://software.intel.com/en-us/articles/how-to-create-self-signed-certificates-for-use-with-intel-sgx-remote-attestation-using
And then register it with the attestation service:
https://software.intel.com/formfill/sgx-onboarding